### PR TITLE
Move lots of update style functions from block_svg to path_object

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Moves more responsibility to the path object.

### Proposed Changes

Move some functions that apply styles to the block's svg root into the path object.

Note that unlike #3390 this doesn't address moving styling into the renderer code.  This just moves the existing addition/removal of CSS styles.

### Reason for Changes
Pre-work for different renderers implementing each of these differently.
Remove the pattern of having lots of different functions that really just decide whether to add or remove a class on the svg root.

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
fyi @BeksOmega @ewpatton

File: core/renderers/common/path_object.js [hunk 1]
```diff
@@ -121,14 +122,33 @@ Blockly.blockRendering.PathObject.prototype.setStyle = function(blockStyle) {
...
 };
 
+/**
+ * Add or remove the given CSS class on the path object's root SVG element.
+ * @param {string} className The name of the class to add or remove
+ * @param {boolean} add True if the class should be added.  False if it should
+ *     be removed.
+ * @protected
+ */
+Blockly.blockRendering.PathObject.prototype.setClass = function(
```
Comment (samelhusseini) [inline core/renderers/common/path_object.js]: underscore in the method name for @protected methods.

**Comment by samelhusseini** [comment]
Compiler complaining that the geras path object also needs to be updated with these changes.

**Comment by BeksOmega** [comment]
LGTM. I really like the setClass addition!

**Pull request merged**